resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/config.scala

```diff
@@ -78,6 +78,20 @@ package object config extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val AM_SEND_TOKEN_CONF =
+    ConfigBuilder("spark.yarn.am.sendTokenConf")
```

Comment:
nit.`spark.yarn.am.tokenConf` instead `spark.yarn.am.sendTokenConf`? `sendTokenConf` sounds like a boolean config like `send` or `not send`.

Reply:
I don't know what is a good name for this and just followed the Hadoop side config name. `send` here is supposed to mean that the token conf is sent from AM to RM

Reply:
Use `regexConf` ? 
Also, add `.regex` to config name ? (in addition to @dongjoon-hyun's suggestion for rename).

Reply:
Thanks, I updated the config name to `spark.yarn.am.tokenConfRegex`. Let me know if this looks better.